tests/collecting/CollectorsTest.py

```diff
@@ -422,3 +424,38 @@ def test_get_all_bears_names(self):
                  'TestDepBearAA',
                  'AspectTestBear',
                  'TestDepBearDependsAAndAA'})
+
+    def test_get_all_languages(self):
+        with bear_test_module():
+            languages = get_all_languages()
+            assert isinstance(languages, tuple)
+            self.assertSetEqual(
+                {str(language) for language in languages},
+                {'C',
+                 'C#',
+                 'CPP',
+                 'CSS',
+                 'Fortran',
+                 'Golang',
+                 'Hypertext Markup Language 2.0, 3.2, 4.0, 4.1, 5, 5.1',
+                 'Java',
+                 'JavaScript',
+                 'JavaServer Pages',
+                 'Jinja2',
+                 'Markdown',
+                 'Matlab',
+                 'ObjectiveC',
+                 'PHP',
+                 'PLSQL',
+                 'PowerShell',
+                 'Python 2.7, 3.3, 3.4, 3.5, 3.6',
+                 'Ruby',
+                 'Scala',
+                 'Shell',
+                 'Swift',
+                 'Tcl',
+                 'TypeScript',
+                 'Unknown',
+                 'Vala',
+                 'VisualBasic',
+                 'm4'})
```

Comment:
This list needs to be loaded from the directory to check all lang files were loaded

Reply:
If I load this from the directory, the versions won't come. It would be `Python` instead of `Python 2.7, 3.3, 3.4, 3.5, 3.6`. Then I have to use `Languages` for that, which would render the test useless.

One solution to check whether all files are loaded or not is to check that the number of language files in the directory are equal to the number of items supplied by `get_all_languages()`.

Reply:
a count comparson sounds good.